src/style.css

```diff
@@ -0,0 +1,49 @@
+html {
+  --star-height: 12px;
+  --star-wigth: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: var(--star-wigth);
+  height: var(--star-height);
+  background-image: url(images/star.svg);
+  background-size: cover;
+  margin: 2px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 6px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
You need to combine all these selectors with a comma so as not to duplicate so much code